ssConfig versus ssConfigFile

Discovered when testing migration from v1 to v2: we should probably only have one of these vars to reduce confusion.

I've just been bitten by this -- I have a folder structure like this:

```
.
├── build.xml
├── _public/
└── static
    └── lib/
        ├── staticSearch/
        │   └── EDITION
        └── staticSearch_config.xml
```

And here is what the build target looks like (with relevant properties):

```xml
  <property name="staticSearch.dir" value="${static.dir}/lib/staticSearch"/>
  <property name="ssConfigFile" value="${static.dir}/staticSearch_config.xml"/>
  <property name="ssVerbose" value="false"/>
  <property name="ssTarget" value="allButValidate"/>

  <target name="search.build" depends="getCommitHash">
    <ant antfile="${staticSearch.dir}/build.xml" 
      target="${ssTarget}" inheritall="false">
      <property name="ssConfigFile" value="${ssConfigFile}"/>
      <property name="ssVerbose" value="${ssVerbose}"/>
    </ant>
  </target>

```

But when I go to run the auto-conversion, I get this:

```
 ant search 
Buildfile: /Users/takeda/projects/wilde/build.xml

getCommitHash:

search.build:

checkConfigVersion:
     [echo] Checking version of config file /Users/takeda/projects/wilde/static/lib/staticSearch/configTest.xml...

convertConfigFile:
    [input] Your config file is for version 1 of staticSearch.
    [input]  It will be converted to version 2 format.
    [input]  Would you like to overwrite the existing file or 
    [input]  create a new file? (o=overwrite, n=new) (o, [n])
o

BUILD FAILED
/Users/takeda/projects/wilde/build.xml:130: The following error occurred while executing this line:
/Users/takeda/projects/wilde/static/lib/staticSearch/build.xml:262: The following error occurred while executing this line:
/Users/takeda/projects/wilde/static/lib/staticSearch/build.xml:285: Warning: Could not find file /Users/takeda/projects/wilde/static/lib/staticSearch/configTest.xml to copy.

Total time: 9 seconds

```

I think the problem comes from the conversion target reconstructing the path (e.g. using `${ssBaseDir.converted}/${ssConfig}` rather than using `ssConfigFile`): 

https://github.com/projectEndings/staticSearch/blob/d1219a37d6f23c9cb68de3d59ad2bc6790e47e63/build.xml#L265-L301

So I think the solution is fairly simple -- always use ssConfigFile rather than ssConfig once we're in the build process.

--- a/build.xml
+++ b/build.xml
@@ -251,7 +251,7 @@
       @version="2" on its root element. If not, it invokes the
       conversion process.
     </description>
-    <echo message="Checking version of config file ${ssBaseDir.converted}/${ssConfig}..."/>
+    <echo message="Checking version of config file ${ssConfigFile}..."/>
     <fileset id="configOnly" dir="${ssBaseDir.converted}" includes="${ssConfig}">
       <containsregexp expression="&lt;config[^&gt;]+version\s*=\s*.2."/>
     </fileset>
@@ -282,15 +282,15 @@ create a new file? (o=overwrite, n=new)"
     <if>
       <equals arg1="${outputStrategy}" arg2="o"/>
       <then>
-        <copy file="${ssBaseDir.converted}/${ssConfig}" tofile="${ssBaseDir.converted}/${ssConfig}.BAK"/>
-        <echo message="A backup of your config file has been created: ${ssBaseDir.converted}/${ssConfig}.BAK"/>
+        <copy file="${ssConfigFile}" tofile="${ssConfigFile}.BAK"/>
+        <echo message="A backup of your config file has been created: ${ssConfigFile}.BAK"/>
       </then>
     </if>
     
     <!--<echo message="${outputStrategy}"/>-->
     <java classpath="${ssSaxon}" classname="net.sf.saxon.Transform" failonerror="true" fork="true">
       <arg value="-xsl:${ssBaseDir.converted}/xsl/convert_v1_to_v2.xsl"/>
-      <arg value="-s:${ssBaseDir.converted}/${ssConfig}"/>
+      <arg value="-s:${ssConfigFile}"/>
       <arg value="--suppressXsltNamespaceCheck:on"/>
       <arg value="ssBaseDir=${ssBaseDir.converted}"/>
       <arg value="ssVerbose=${ssVerbose}"/>

